### DISCUSSION TITLE
Move "Configuration" in header menu to last position

### DISCUSSION THREAD

**Discussion by bnhf** [bnhf/openvpn-admin-plus#26] [Q&A]
@karabelnikov 

I'm thinking our menu header should be rearranged -- with Configuration last instead of second. Once the config has been done the first time it's the least likely to be used. If that sounds good, would you mind taking that on? Should be an easy one I would guess. :-)

**Comment by bnhf** [comment]
@karabelnikov 

I guessing you missed this question in "Discussions" -- let me know what you think when you have a chance.

**Comment by karabelnikov** [comment]
The transfer of the section, these are cosmetic changes. Yes, I agree that the configuration section is not used so often, but on the other hand, the certificates section is also used. There is more on color and taste, all markers are different)) Do you think it makes sense?
